stylesheets/_component.panels.scss

```diff
@@ -19,6 +19,7 @@
 
     &__body {
         max-height: 20em;
+        overflow-y: auto;
```

Comment:
I think we should add an additional option to `html.panel` for this behaviour, `scroll` maybe? That could be used to add a overflow modifier class for the scroll and also to add `tabindex="0" to the panel body, otherwise there is no way for a keyboard only user to scroll the content of the overflowing panel body (unless they are using Firefox, which I think is the only browser that currently makes block level elements with an overflow focusable via the keyboard).